backend/grants/admin.py

```diff
@@ -443,14 +444,15 @@ class GrantAdmin(ExportMixin, ConferencePermissionMixin, admin.ModelAdmin):
         "status",
         "country_type",
         "occupation",
-        "grant_type",
+        "approved_type",
```

Comment:
"grant_type" (Speaker | Diversity | Unemployed) don't think we actually use it as now there are the more precise filters "IsProposedSpeakerFilter". 
There was missing instead `approved_type` that is useful instead.

Reply:
This is used during reviews I think?

Reply:
mmm not really if I remember correctly... And now there is the review system so...